src/graph/transform/remove_edges.cc

```diff
@@ -0,0 +1,88 @@
+/*!
+ *  Copyright (c) 2019 by Contributors
+ * \file graph/transform/remove_edges.cc
+ * \brief Remove edges.
+ */
+
+#include <dgl/base_heterograph.h>
+#include <dgl/transform.h>
+#include <dgl/array.h>
+#include <dgl/packed_func_ext.h>
+#include <dgl/runtime/registry.h>
+#include <dgl/runtime/container.h>
+#include <vector>
+#include <utility>
+#include <tuple>
+
+namespace dgl {
+
+using namespace dgl::runtime;
+using namespace dgl::aten;
+
+namespace transform {
+
+std::pair<HeteroGraphPtr, std::vector<IdArray>>
+RemoveEdges(const HeteroGraphPtr graph, const std::vector<IdArray> &eids) {
+  std::vector<IdArray> induced_eids;
+  std::vector<HeteroGraphPtr> rel_graphs;
+  const int64_t num_etypes = graph->NumEdgeTypes();
+
+  for (int64_t etype = 0; etype < num_etypes; ++etype) {
+    const SparseFormat fmt = graph->SelectFormat(etype, SparseFormat::kAny);
```

Comment:
This is a tricky. Actually, we want to pick whatever format that has consecutive eid array.

Reply:
How do you recommend to do this?  Should I create another function called `SelectFormatWithConsecutiveID` or something like that?

Reply:
Let's put `SparseFormat::kCOO` as the preferred format for now as it usually has consecutive ids.